grafana-plugin/src/containers/IntegrationLabelsForm/IntegrationLabelsForm.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useCallback, useState } from 'react';
+
+import { Button, Drawer, HorizontalGroup, Icon, InlineSwitch, Input, Label, Tooltip, VerticalGroup } from '@grafana/ui';
+import cn from 'classnames/bind';
+import { observer } from 'mobx-react';
+
+import { AlertReceiveChannel } from 'models/alert_receive_channel/alert_receive_channel.types';
+import { LabelKey } from 'models/label/label.types';
+import { useStore } from 'state/useStore';
+
+import styles from './IntegrationLabelsForm.module.css';
+
+const cx = cn.bind(styles);
+
+interface IntegrationLabelsFormProps {
+  id: AlertReceiveChannel['id'];
+  onSubmit: () => void;
+  onHide: () => void;
+}
+
+const IntegrationLabelsForm = observer((props: IntegrationLabelsFormProps) => {
+  const { id, onHide, onSubmit } = props;
+
+  const store = useStore();
+
+  const { alertReceiveChannelStore } = store;
+
+  const alertReceiveChannel = alertReceiveChannelStore.items[id];
+
+  const [alertGroupLabels, setAlertGroupLabels] = useState(alertReceiveChannel.alert_group_labels);
+
+  const handleSave = useCallback(() => {
+    alertReceiveChannelStore.saveAlertReceiveChannel(id, { alert_group_labels: alertGroupLabels });
+
+    onSubmit();
+
+    onHide();
+  }, [alertGroupLabels]);
```

Comment:
Why use `useCallback` for such simple usages?

https://kentcdodds.com/blog/usememo-and-usecallback#why-is-usecallback-worse

Reply:
I think it's the most famous article about hooks in the web :D